compiler/rustc_driver/Cargo.toml

```
@@ -38,4 +38,4 @@ winapi = { version = "0.3", features = ["consoleapi", "debugapi", "processenv"]
 
 [features]
 llvm = ['rustc_interface/llvm']
-release_max_level_info = ['tracing/release_max_level_info']
+release_max_level_info = ['tracing/release_max_level_info', 'tracing/max_level_info']
```

Does this need both? I think we could just set `tracing/max_level_info`.

Similarly I think this option should be renamed to `max_level_info` in `rustc_driver`.
```suggestion
max_level_info = ['tracing/max_level_info']
```